script/install/debootstrap.bash

```diff
@@ -131,6 +136,13 @@ chroot "$ISOLATE_ROOT" apt-get install openjdk-11-jdk # Java
   chroot "$ISOLATE_ROOT" apt-get install python3.8 # Python 3.8
   # note: when updating these Python versions, also update the check for adding the PPA above
 
+  # PyPy https://www.pypy.org
+  # Python 3.6 is the latest version of Python compatible with pypy
+  # https://launchpad.net/~pypy/+archive/ubuntu/ppa (added earlier)
```

Comment:
```suggestion
  # PyPy
  # https://www.pypy.org
```
I think the other comments are excessive (the version will just become out of date; the PPA homepage link is redundant because the PPA location is given above, and we don't have homepage links for the other PPAs).